hoomd/snapshot.py

```diff
@@ -31,6 +27,7 @@ def box(self, box):
             raise ValueError(
```

Comment:
On the line above this, I would recommend catching a limited set of exceptions that we know can be thrown by `from_box`.

Reply:
There is a
```
except:
     raise
````
in the method like in `freud` so technically we don't know what errors will be raised necessarily. I generally agree with the limited catching of exceptions, but without converting the error in `from_box` to a `ValueError` I don't think we can do this completely safely.

Reply:
Ok, sounds fine.